packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -61,14 +61,8 @@ export function startRumSessionManager(configuration: RumConfiguration, lifeCycl
         id: session.id,
         plan,
         sessionReplayAllowed: plan === RumSessionPlan.WITH_SESSION_REPLAY,
-        longTaskAllowed:
-          configuration.trackLongTasks !== undefined
-            ? configuration.trackLongTasks
-            : configuration.oldPlansBehavior && plan === RumSessionPlan.WITH_SESSION_REPLAY,
-        resourceAllowed:
-          configuration.trackResources !== undefined
-            ? configuration.trackResources
-            : configuration.oldPlansBehavior && plan === RumSessionPlan.WITH_SESSION_REPLAY,
+        longTaskAllowed: configuration.trackLongTasks,
+        resourceAllowed: configuration.trackResources,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ we could remove those two properties from `RumSessionManager`, and replace their usages with `RumConfiguration` directly.